NotesApi.Tests/V1/E2ETests/Steps/GetNotesSteps.cs

```diff
@@ -67,6 +88,8 @@ private static async Task<PagedResult<NoteResponseObject>> ExtractResultFromHttp
             return apiResult;
         }
 
+        #region When
```

Comment:
What are these for?

Reply:
Collapseable code, so that it doesn't look too convoluted - you can expand/minimize when changing the file.